src/main/resources/templates/js/trends-chart.js.vm

```diff
@@ -1,20 +1,23 @@
+#macro(stringArray $array) [#foreach($element in $array) "$element", #end] #end
+#macro(numberArray $array) [#foreach($element in $array) $element, #end] #end
+
```

Comment:
That is architecturally not a sensible decision. The template engine is responsible for transforming the data into the output format and knows what context it will be injected into. Sidestepping this by using java adds complexity as java does not know the output format or any restrictions that apply such as the need to escape characters.

I will look into a test to see if the correct lines are output into overview-trends.html but frankly said, this part of the code is not setup for automated testing and will always need a manual test.